project-base/tests/ReadModelBundle/Functional/Image/ImageViewRepositoryTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\ReadModelBundle\Functional\Image;
+
+use Shopsys\FrameworkBundle\Model\Product\Product;
+use Shopsys\ReadModelBundle\Image\ImageViewInterface;
+use Shopsys\ReadModelBundle\Image\ImageViewRepository;
+use Tests\ShopBundle\Test\FunctionalTestCase;
+
+class ImageViewRepositoryTest extends FunctionalTestCase
+{
+    public function testGetForSingleEntityId(): void
+    {
+        $testedProductId = 1;
+
+        $imageViewRepository = $this->getContainer()->get(ImageViewRepository::class);
+
+        $imageViews = $imageViewRepository->getForEntityIds(Product::class, [$testedProductId]);
+
+        $this->assertCount(1, $imageViews);
```

Comment:
this would be definitelly more readable if you changed it to the `$expected = ....; $this->assertEquals($expected, $imageViews)` format